#!/usr/bin/env python
PACKAGE = "vqimg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


gen.add("hue",     double_t, 0, "Hue",                                      .00, 0, 1)
gen.add("min_sat", double_t, 0, "Minimal saturation required",              .50, 0, 1)
gen.add("min_val", double_t, 0, "Minimal intensity value required",         .50, 0, 1)


exit(gen.generate(PACKAGE, "vqimg", "HueParam"))
